src/resources/parser/material/json-standard-material.js

```diff
@@ -14,11 +14,67 @@ Object.assign(pc, function () {
         var validated = this._validate(migrated);
 
         var material = new pc.StandardMaterial();
-        material.initialize(validated);
+        this.initialize(material, validated);
 
         return material;
     };
 
+    /**
+     * @private
+     * @function
+     * @name pc.JsonStandardMaterialParser#initialize
+     * @description  Initialize material properties from the material data block e.g. loading from server
+     * @param {pc.StandardMaterial} material The material to be initialized
+     * @param {Object} data The data block that is used to initialize
+     */
+    JsonStandardMaterialParser.prototype.initialize = function (material, data) {
+        // usual flow is that data is validated in resource loader
+        // but if not, validate here.
+        if (!data.validated) {
+            if (!material._validator) {
+                material._validator = new pc.StandardMaterialValidator();
```

Comment:
The parser already has an internal instance of a validator, presumably we can use this.

Reply:
I've switched to using the internal instance. Look good?